scripts/runtests.py

```diff
@@ -44,16 +45,28 @@ def _get_strict_params(stub_path: str) -> list[str]:
 
 
 def main() -> None:
-    try:
-        path = sys.argv[1]
-    except IndexError:
-        print("Missing path argument in format <folder>/<stub>", file=sys.stderr)
-        sys.exit(1)
-    assert os.path.exists(path), rf"Path {path} does not exist."
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--run-stubtest",
+        action="store_true",
+        help="Run stubtest for the selected package(s). Running stubtest may download and execute arbitrary code from PyPI: "
+        + "only use this option if you trust the package you are testing.",
```

Comment:
Minor formatting nit:

```suggestion
        help=(
            "Run stubtest for the selected package(s). Running stubtest may download and execute arbitrary code from PyPI: "
            "only use this option if you trust the package you are testing."
        ),
```